custom_components/blitzortung/__init__.py

```diff
@@ -86,38 +86,30 @@ async def async_setup_entry(hass: HomeAssistant, config_entry: ConfigEntry):
         server_stats=config.get(const.SERVER_STATS),
     )
 
-    hass.data[DOMAIN][config_entry.entry_id] = coordinator
-
     await hass.config_entries.async_forward_entry_setups(config_entry, PLATFORMS)
-    await coordinator.connect()
+    await config_entry.runtime_data.connect()
 
     if not config_entry.update_listeners:
         config_entry.add_update_listener(async_update_options)
 
     return True
 
 
-async def async_update_options(hass, config_entry):
+async def async_update_options(hass, config_entry: BlitzortungConfigEntry):
     """Update options."""
     _LOGGER.info("async_update_options")
     await hass.config_entries.async_reload(config_entry.entry_id)
 
 
-async def async_unload_entry(hass: HomeAssistant, config_entry: ConfigEntry):
+async def async_unload_entry(hass: HomeAssistant, config_entry: BlitzortungConfigEntry):
     """Unload a config entry."""
-    coordinator: BlitzortungCoordinator = hass.data[DOMAIN][config_entry.entry_id]
-    await coordinator.disconnect()
+    await config_entry.runtime_data.disconnect()
     _LOGGER.debug("Disconnected")
 
-    # Cleanup platforms.
-    if unload_ok := await hass.config_entries.async_unload_platforms(config_entry, PLATFORMS):
-        # Pop coordinator only when unload was succesfull.
-        hass.data[DOMAIN].pop(config_entry.entry_id)
```

Comment:
We don't want to remove `config` because reloading the config entry doesn't load the yaml configuration.